app/services/indicator_service.js

```diff
@@ -200,12 +200,38 @@ const getTags = (scorecardUuid) => {
           .filter((tag, index, self) => self.indexOf(tag) == index);
 }
 
+const getIndicatorsState = (props, state) => {
```

Comment:
@limkimsan please kindly refactor to move the logic of the form component to locate in itself appropriately.